app/views/comments/_comments.html.slim

```diff
@@ -0,0 +1,13 @@
+  #comments
+    h2 Comments
+
+    - commentable.comments.each do |comment|
+      hr(id="#{dom_id comment}")
+      h5 = "#{ user_for_comment(comment) }, #{ l(comment.created_at, format: :short) }".html_safe
```

Comment:
Generally in favor, but I would do this in a separate PR. (Also affects all sorts of parts of the app.